command.go

```diff
@@ -411,6 +423,18 @@ func (c *Command) HelpTemplate() string {
 {{end}}{{if or .Runnable .HasSubCommands}}{{.UsageString}}{{end}}`
 }
 
+// VersionTemplate return version template for the command.
+func (c *Command) VersionTemplate() string {
+	if c.versionTemplate != "" {
+		return c.versionTemplate
+	}
+
+	if c.HasParent() {
+		return c.parent.VersionTemplate()
+	}
+	return `{{with .Use}}{{printf "%s " .}}{{end}}{{printf "version %s" .Version}}`
```

Comment:
Why do we need here `.Use`? 
Why do you think about printing only a version? Just `{{.Version}}`.

Reply:
Can certainly do that as well, but for many applications the version output takes the form `{{appName}} version {{version}}`, so thought that it would be a sensible default. Examples:

```
➜ go version
go version go1.9.2 darwin/amd64
➜ git --version
git version 2.6.3
```

Reply:
Okay. But then use `.Name` instead of `.Use`, because `.Use` can be in format `command [path] [flags]`.

Reply:
Done